src/api/migrations.ts

```diff
@@ -124,7 +124,9 @@ export const migrateRepo = async (repoId: string): Promise<void> => {
           })
         } catch (err) {
           // Silence conflict errors
-          if (err.error !== 'conflict') {
+          if (err.error === 'conflict') {
+            console.log('Migration failed due to conflict:', err)
```

Comment:
Sorry to be nitpicky, but technically the migration didn't fail. It was already done by a different process. Maybe change the error string to "Conflict migrating repo ID ${repoId}. Migration was already completed by another process."